website/lib/build.js

```diff
@@ -16,11 +16,17 @@ fs.copySync('../dist/katex.min.js', 'static/static/katex.min.js');
 fs.copySync('../dist/katex.min.css', 'static/static/katex.min.css');
 fs.copySync('../dist/fonts', 'static/static/fonts');
 
+// copy local built KaTeX extensions
+fs.copySync('../dist/contrib/copy-tex.css', 'static/static/copy-tex.css');
+fs.copySync('../dist/contrib/copy-tex.min.js', 'static/static/copy-tex.min.js');
+fs.copySync('../dist/contrib/mhchem.min.js', 'static/static/mhchem.min.js');
+
+
 // use KaTeX from CDN on the main page for Netlify production deploy
 if (process.env.CONTEXT === 'production') {
     const version = require('../versions.json')[0];
     let indexHtml = fs.readFileSync('pages/index.html', 'utf8');
-    indexHtml = indexHtml.replace(/(["'])static\/(katex|fonts)/g,
+    indexHtml = indexHtml.replace(/(["'])static\/(katex|fonts|copy-tex|mhchem)/g,
```

Comment:
You'll need a separate `.replace()` here, because the correct CDN URL for plugins is slightly different. For mhchem, it is 
`https://cdn.jsdelivr.net/npm/katex@0.10.1/dist/contrib/mhchem.min.js`
as explained here:
https://github.com/KaTeX/KaTeX/tree/master/contrib/mhchem

Reply:
Included these changes.